publish.ps1

```diff
@@ -68,10 +68,12 @@ function do-package()
     Copy-Item -Force "$distDir\readme.txt"            $tempDir -ErrorAction Stop
 
     # Create archive
-    7za a -tzip -mx9 $outZip "$tempDir\*"
+    7z a -tzip -mx9 $outZip "$tempDir\*"
```

Comment:
No, 7z is the file extension it normally uses.  I've never seen a 7za file, and if you have 7zip it looks primarily for 7z files.